localgov_alert_banner.links.task.yml

```diff
@@ -16,14 +16,19 @@ entity.localgov_alert_banner.version_history:
   title: 'Revisions'
 
 entity.localgov_alert_banner.status_form:
-  route_name:  entity.localgov_alert_banner.status_form
-  base_route:  entity.localgov_alert_banner.canonical
+  route_name: entity.localgov_alert_banner.status_form
+  base_route: entity.localgov_alert_banner.canonical
   title: 'Status'
   weight: 10
   class: '\Drupal\localgov_alert_banner\Plugin\Menu\LocalTask\StatusFormTab'
 
 entity.localgov_alert_banner.delete_form:
-  route_name:  entity.localgov_alert_banner.delete_form
-  base_route:  entity.localgov_alert_banner.canonical
+  route_name: entity.localgov_alert_banner.delete_form
+  base_route: entity.localgov_alert_banner.canonical
   title: Delete
   weight: 10
+
+entity.localgov_alert_banner.collection:
+  route_name: view.localgov_admin_manage_alert_banners.localgov_alert_banner_admin_list
```

Comment:
Can we change the route name to `entity.localgov_alert_banner.collection`
Theres a weird bug that when you go to the alert banner page the tabs no longer show. I think its to do with defining a collection route in the entity class, but we don't use the collection, we use the view. This causes issues with the menu / tab display but we will sort these out in another issue.